### PR TITLE
pass logger config to mockserver

### LINKED ISSUE #363: Pass logger config to mockserver
Configuration for mockserver is hardcoded in https://github.com/celestiaorg/optimint/blob/tzdybal/latest-cosmos/da/grpc/mockserv/mockserv.go#L18.

It would be great to have a possibility to pass this configuration.

### PR DESCRIPTION
**Pull request by randomshinichi**
closes #363

### CODE REVIEW

**Review by tzdybal — APPROVED** [review]
Thanks for your submission!

Usually logger is passed as last argument in our codebase, but it doesn't really matter.

**Comment by randomshinichi** [comment]
I can do that... just pushed

File: da/grpc/mockserv/cmd/main.go [hunk 1]
```diff
@@ -24,7 +27,7 @@ func main() {
 		log.Panic(err)
 	}
 	log.Println("Listening on:", lis.Addr())
-	srv := mockserv.GetServer(kv, conf, nil)
+	srv := mockserv.GetServer(kv, conf, nil, logger)
```
Comment (tzdybal) [inline da/grpc/mockserv/cmd/main.go:30]: :+1:

File: da/grpc/mockserv/cmd/main.go [hunk 2]
```diff
@@ -4,15 +4,18 @@ import (
 	"flag"
 	"log"
 	"net"
+	"os"
 	"strconv"
 
 	grpcda "github.com/celestiaorg/rollmint/da/grpc"
 	"github.com/celestiaorg/rollmint/da/grpc/mockserv"
 	"github.com/celestiaorg/rollmint/store"
+	tmlog "github.com/tendermint/tendermint/libs/log"
```
Comment (tzdybal) [inline da/grpc/mockserv/cmd/main.go:13]: This is mixing "local" and 3rd party imports. `tmlog` should be imported in separate "section" before all the rollmint imports.

This should be detected by linter.

**Review by tzdybal — COMMENTED** [review]
Thanks for taking a time to fix the nit-pick comment.

There is one tiny issue with imports, I'll update linter to automatically catch such cases.

**Pull request merged**